frontend/src/components/Menu.vue

```diff
@@ -0,0 +1,27 @@
+<!-- TODO: Extract all unnecessary logic from App.vue to a separate view -->
+
+<script setup>
+const submitForm = () => {
+  console.log("Form data submitted!");
+};
+
+const exportToPDF = () => {
+  print();
+};
+</script>
+
+<template>
+  <nav>
+    <button class="save-btn" @click="exportToPDF">Export to PDF</button>
```

Comment:
```suggestion
    <button class="save-btn" @click="window.print()">Export to PDF</button>
```

hope this works. **Simple is better than complex**

Reply:
I tried it, but it didn't work, maybe it's not that bad just because it's easier to extend the functionality if I need

Reply:
got it. yep, correct, see that it doesn't work